gremlin-go/driver/client_test.go

```diff
@@ -21,7 +21,12 @@ package gremlingo
 
 import (
 	"crypto/tls"
+	"fmt"
+	"github.com/google/uuid"
 	"github.com/stretchr/testify/assert"
+	"gopkg.in/yaml.v3"
```

Comment:
What is license type?

Reply:
It is MIT, it seems like just about all YAML parsing libraries use MIT license. I will update the PR soon with proper updates to LICENSE and NOTICE files.

Reply:
The yaml dependency here or in any of the other GLVs is not bundled in any distributions of tinkerpop so no updates to LICENSE or NOTICE files are required.